jscomp/others/js_json.mli

```diff
@@ -214,8 +214,7 @@ Js.log(Js.Json.stringifyWithSpace(Js.Json.object_(dict), 2))
 ```
 *)
 
-external stringifyAny : 'a -> string option = "stringify"
-  [@@bs.val] [@@bs.scope "JSON"]
+val stringifyAny : 'a -> string option
```

Comment:
Hmm, if I understand this correctly, we're not exposing the zero cost function anymore, instead swapping it out to something with a runtime.

Given that, I think we should also expose `stringifyAnyExn`, and make clear in the release notes that one can switch to that if the old behavior is to be preserved. What do you think?

Reply:
Good point. I agree.

I'm afraid of getting off-topic, what is the context of not exposing the zero cost function anymore?

Reply:
Not sure I understand correctly.
Is `serializeExn` one example function which is swapping out to something with runtime? If so, `stringifyAnyExn` would not need, because the `serializeExn` would be a better replacement.

Reply:
@zth If `serializedExn` is one of example replacing the zero cost function, then `stringifyAnyExn` doesn't need to be exposed, I think. What do you think?

Reply:
Sorry, I'll reply here soon, out traveling. Haven't forgotten, promise!

Reply:
Sorry, interupting you. No need to hurry at all. Have a good travel.